spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "GET #show" do
+    it "returns http success" do
+      get :show
+      expect(response).to have_http_status(:success)
```

Comment:
remember, the trickiest part of testing is setting up your data correctly and making sure you're testing the right "thing".  besides the expected response, what is the biggest difference between these 2 tests?